src/sentry/static/sentry/app/views/organizationStream/overview.jsx

```diff
@@ -417,10 +449,17 @@ const OrganizationStream = createReactClass({
     let {orgId} = this.props.params;
     let access = this.getAccess();
 
-    // In the project mode this reads from the project feature.
-    // There is no analogous property for organizations yet.
+    // If we have a selected project we can get release data
     let hasReleases = false;
-    let latestRelease = '';
+    let projectId = null;
+    let latestRelease = null;
+    let {selectedProject} = this.state;
+    if (selectedProject) {
+      let features = new Set(selectedProject.features);
```

Comment:
Can we use the `<Feature>` component here? I think it generally grabs project from context, but it will take project from props.

Reply:
Possible. I can circle back to that as it requires more changes in components shared on the current issue stream as well.